qemu/tests/cpu_model.py

```diff
@@ -0,0 +1,85 @@
+import logging
+
+from virttest import error_context
+from virttest import env_process
+from virttest import utils_misc
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Qemu reboot test:
+    1) Start qemu to get cpu model supported by host
+    3) Boot guest with the cpu model
+    4) Check cpu model name in guest
+    5) Check cpu flags in guest(only for linux guest)
+    6) Reboot guest
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    error_context.context("Start qemu to get support cpu model", logging.info)
+    vm = env.get_vm(params["main_vm"])
+    out = vm.monitor.info("cpu-definitions")
+    vm.destroy()
+    model = params["model"]
+    model_pattern = params["model_pattern"]
+    flags = params["flags"]
+    if utils_misc.get_cpu_vendor(verbose=False) == 'GenuineIntel':
+        model_ib = "%s-IBRS" % model
+        flag_ib = " ibpb ibrs"
+        name_ib = ", IBRS"
+    else:
+        model_ib = "%s-IBPB" % model
+        flag_ib = " ibpb"
+        name_ib = "(with IBPB)"
+
+    model_list = list(filter(lambda x: not x["unavailable-features"], out))
```

Comment:
better to use list comprehension

Reply:
Can you explain why, though I've updated.

Reply:
list comprehension is faster.